index.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const path = require('path')
+
+const send = require('send')
+
+const fp = require('fastify-plugin')
+
+function fastifyStatic (fastify, opts, next) {
+  if (typeof opts.sendOptions !== 'object') {
+    return next(new Error('"sendOptions" option is required'))
+  }
+  if (typeof opts.sendOptions.root !== 'string') {
+    return next(new Error('"sendOptions.root" option is required'))
+  }
+  if (!path.isAbsolute(opts.sendOptions.root)) {
+    return next(new Error('"directory" option must be an absolute path'))
+  }
+  if (typeof opts.redirectionLink404 !== 'string') {
+    return next(new Error('"redirectionLink404" option is required'))
+  }
+  if (typeof opts.redirectionLink403 !== 'string') {
+    return next(new Error('"redirectionLink403" option is required'))
+  }
+  if (typeof opts.redirectionLink500 !== 'string') {
+    return next(new Error('"redirectionLink500" option is required'))
+  }
```

Comment:
I think we should have good defaults for all of these, the developer experience should require just the root folder as an option.

Plus, I would not have `sendOptions` at all, but just options for this plugin.